### DISCUSSION TITLE
Internet connection required to use LAN models

### DISCUSSION THREAD

**Discussion by eort** [lnccbrown/HSSM#256] [Q&A] [answered]
Hi everyone, 

HSSM looks really super nice! I wanted to try out a few bigger models on our HPC, which has no direct connection to the internet. Unfortunately, when I try to define a model that is based on a LAN likelihood, the lack of internet connection is a problem:

`LocalEntryNotFoundError: Connection error, and we cannot find the requested files in the disk cache. Please try again or make sure your Internet connection is on.
`
Based on the error message, it sounds like I could bypass the problem, by moving the file into the disk cache? Is that so, and if yes, how would I do that?

Thanks!

Eduard

**Comment by digicosmos86** [comment, marked as answer]
Hi Eduard,

Glad to hear that you are enjoying using HSSM! Yes, we download the approximate differentiable likelihoods as needed, which requires the internet connection. You can download any and all models yourself with the method described [here](https://github.com/lnccbrown/HSSM/issues/248)

Paul

**Reply by eort** [reply]
Very nice! Thanks
